src/GNDStk/Node/src/special.hpp

```diff
@@ -150,7 +150,7 @@ const std::string &comment(
 ) const {
    size_t count = 0;
 
-   for (auto c : children)
+   for (auto &c : children)
```

Comment:
Been there, done that

Reply:
And in this case it wasn't just an issue of inefficiency from copying. The code (if instantiated!) didn't compile, because those things are unique pointers - which don't copy!